src/main/scala/cognite/spark/v1/DefaultSource.scala

```diff
@@ -398,6 +398,9 @@ object DefaultSource {
     val collectMetrics = toBoolean(parameters, "collectMetrics")
     val collectTestMetrics = toBoolean(parameters, "collectTestMetrics")
 
+    val enableSinglePartitionDeleteAssetHierarchy =
+      toBoolean(parameters, "enableSinglePartitionDeleteHierarchy")
```

Comment:
```suggestion
      toBoolean(parameters, "enableSinglePartitionDeleteHierarchy", defaultValue = false)
```
maybe even like this, though it is the default default